OPAL/tac/src/main/scala/org/opalj/tac/fpcf/analyses/cg/AndroidICCAnalysis.scala

```diff
@@ -830,57 +845,65 @@ class AndroidICCAnalysis(val project: SomeProject) extends FPCFAnalysis {
         } else implicitIntents += intent
     }
 
+    def getClassFileFromParameter(p: Expr[DUVar[ValueInformation]], tacCode: TACode[TACMethodParameter, DUVar[ValueInformation]]): Option[ClassFile] = {
+        val defSite = p.asVar.definedBy.head
+        if (defSite > -1) {
+            try {
+                project.classFile(tacCode.stmts(defSite).asAssignment.expr.asClassConst.value.asObjectType)
+            } catch {
+                case _: Throwable ⇒ None
```

Comment:
Looks fishy, please write your code in such way that it doesn't cause an exception in the first place.